### PR TITLE
Feature/ Dataset Details: Add Test Reports page

### PR DESCRIPTION
**Pull request by Leshe4ka**

### CODE REVIEW

File: odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewDataQualityReport/OverviewDataQualityReport.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    <div className={classes.container}>
+      <Grid container direction="column">
+        <Grid item container wrap="nowrap" justify="space-between">
+          <Grid item>
+            <Typography variant="h4">Test report</Typography>
+          </Grid>
+          <Grid item>
+            <AppButton size="small" color="tertiary" onClick={() => {}}>
+              {/* Add path to test reports page */}
+              <Link to="/">See all</Link>
```
Comment (soffest) [inline odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewDataQualityReport/OverviewDataQualityReport.tsx:44]: Please add correct link here.

File: odd-platform-ui/src/components/DataEntityDetails/TestReport/TestReportItem/TestReportDetails/TestReportDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  DataQualityApiGetRunsRequest,
+  DataQualityTestRun,
+} from 'generated-sources';
+import { Grid } from '@material-ui/core';
+import { StylesType } from 'components/DataEntityDetails/TestReport/TestReportItem/TestReportDetails/TestReportDetailsStyles';
```
Comment (soffest) [inline odd-platform-ui/src/components/DataEntityDetails/TestReport/TestReportItem/TestReportDetails/TestReportDetails.tsx:7]: Would be better to use relative path here.

File: odd-platform-ui/src/components/DataEntityDetails/TestReport/TestReportItem/TestReportItem.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+} from 'components/shared/TestReportTypeItem/TestReportTypeItem';
+import cx from 'classnames';
+import MinusIcon from 'components/shared/Icons/MinusIcon';
+import PlusIcon from 'components/shared/Icons/PlusIcon';
+import TestItem from 'components/DataEntityDetails/TestReport/TestReportItem/TestItem/TestItem';
+import { dataEntityTestPath } from 'lib/paths';
+import { Link } from 'react-router-dom';
+import { StylesType } from './TestReportItemStyles';
+
+interface TestReport {
```
Comment (soffest) [inline odd-platform-ui/src/components/DataEntityDetails/TestReport/TestReportItem/TestReportItem.tsx:15]: You have very similar interface in another component. Please consider moving it to redux/interfaces and reusing where necessary.

File: odd-platform-ui/src/components/shared/TestReportTypeItem/TestReportTypeItem.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core';
+import cx from 'classnames';
+import {
+  styles,
+  StylesType,
+} from 'components/shared/TestReportTypeItem/TestReportTypeItemStyles';
+
+export type DataSetTestReportTypeNames =
```
Comment (soffest) [inline odd-platform-ui/src/components/shared/TestReportTypeItem/TestReportTypeItem.tsx:9]: Very similar to DataQualityTestRunStatusEnum that is being generated from api spec.

**Comment by DementevNikita** [comment]
Merged in another PR

**Pull request closed without merging**